src/main/webapp/assets-lti/js/factories/factories.js

```diff
@@ -114,8 +114,15 @@ canvasSupportApp.factory('Course', function ($http) {
 //SECTIONS FACTORY - does the request for the sections controller
 canvasSupportApp.factory('Sections', function ($http) {
   return {
-    getSectionsForCourseId: function (courseId) {
-      var url = '/canvasCourseManager/manager/api/v1/courses/' + courseId + '/sections?per_page=100&_='+ generateCurrentTimestamp();
+    getSectionsForCourseId: function (courseId, context) {
+    	console.log('Course: ' + courseId);
+    	console.log('Context: ' + context);
+    	if ( context === true){
```

Comment:
I would add a comment why handling this request is different LTI vs Service desk calls